tests/index.js

```diff
@@ -38,8 +38,10 @@ describe('configurations', function() {
       assert.equal(configName.indexOf('react/'), 0);
       assert.equal(plugin.configs.all.rules[configName], 2);
     });
-    rules.forEach(function(ruleName) {
-      assert(plugin.configs.all.rules['react/' + ruleName]);
+    ruleFiles.forEach(function(ruleName) {
+      var inDeprecatedRules = Boolean(plugin.deprecatedRules[ruleName]);
```

Comment:
I'm happy to change, but it looked like in this project `Boolean` (12 occurrences) was preferred over `!!` (0 occurrences)